Makefile

```diff
@@ -46,6 +46,29 @@ test: generate fmt vet manifests
 manager: generate fmt vet
 	go build -o bin/manager ./cmd/operator/
 
+manager-amd64: export GOOS=linux
+manager-amd64: export GOARCH=amd64
+manager-amd64: generate fmt vet
+	go build -o bin/manager-amd64 ./cmd/operator/
+
+WEBHOOK_NS=dynatrace
+
+debug-webhook-setup:
+	@echo Preparing webhook deployment for debugging
+	kubectl -n $(WEBHOOK_NS) patch deploy dynatrace-webhook -p '{"spec":{"template":{"spec":{"containers":[{"name":"webhook", "image":"quay.io/gkrenn/testoperator:operatorwithtar","command":["/bin/bash", "-c", "sleep 999999999"]}]}}}}'
```

Comment:
Please don't use images from your private quay repos here

Reply:
sleep 999999999 ?

Reply:
I'll move it to our dynatrace-operator quay repo with a special tag

sleep is used as a placeholder application to prevent the container from restarting.
If we wouldn't provide a new command it would start the webhook provided in the image.